pandas/tests/test_nanops.py

```diff
@@ -888,6 +888,19 @@ def test_nanstd_roundoff(self):
             result = data.std(ddof=ddof)
             assert result == 0.0
 
+    def test_nanvar_timedelta(self):
```

Comment:
I know this *should* be done up in `TestnanopsDataFrame` but I cannot figure out what's going on in those tests. `test_nanstd` and `test_nanvar` both have `allow_tdelta=True`, but were not catching the bug in https://github.com/pandas-dev/pandas/issues/18880